Example/Controllers/SwiftUI/Helper/RemoteImageView.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+
+class ImageLoaderService: ObservableObject {
+    @Published var image: UIImage = UIImage()
+
+    func loadImage(for urlString: String) {
+        guard let url = URL(string: urlString) else { return }
+
+        let task = URLSession.shared.dataTask(with: url) { data, _, _ in
+            guard let data = data else { return }
+            DispatchQueue.main.async {
+                self.image = UIImage(data: data) ?? UIImage()
+            }
+        }
+        task.resume()
+    }
+}
+
+struct RemoteImageView: View {
+    var urlString: String
+    @ObservedObject var imageLoader = ImageLoaderService()
+    @State var image: UIImage = UIImage()
+
+    var body: some View {
+        Image(uiImage: image)
+            .resizable()
+            .aspectRatio(contentMode: .fit)
+            .onReceive(imageLoader.$image) { image in
+                self.image = image
+            }
+            .onAppear {
+                imageLoader.loadImage(for: urlString)
+            }
+    }
+}
+
+struct RemoteImageView_Previews: PreviewProvider {
+    static var previews: some View {
+        RemoteImageView(urlString: "https://via.placeholder.com/300.png")
```

Comment:
Can we use someother placholder image from apple ?

Reply:
sure I'll replaced it with a example image from `developer.apple.com` url for now

Reply:
@laubengaier @rleojoseph you can use the [SDK team](https://github.com/orgs/rakutentech/teams/miniapp-sdk-ios) logo: 
https://avatars.githubusercontent.com/t/6292214

Reply:
replaced it with the avatar

can now be previewed like so:
<img width="482" alt="Screen Shot 2022-09-27 at 11 05 18" src="https://user-images.githubusercontent.com/17570451/192414604-35ad7332-ba13-4ffc-8bf4-ef9bd310fbe7.png">